### ISSUE TITLE
API returns unexpected results.

### ISSUE THREAD

**Issue by ermaker** [cloudatcost/api#39]
Some APIs return errors.

For example:

tasks:

```
$ curl -k "https://panel.cloudatcost.com/api/v1/listtasks.php?key=[KEY]&login=[LOGIN]"
Unknown column 'iod.apitask.cid' in 'field list'
$
```

delete:

```
$ curl -k --data "key=[KEY]&login=[LOGIN]&sid=[SID]" https://panel.cloudatcost.com/api/v1/cloudpro/delete.php
Table 'panel.hardware_hd' doesn't exist
$
```

Any ideas for this situation?

**Comment by stephenwithav** [comment]
This is an issue on their end, so no fixes yet.

**Comment by ermaker** [comment]
Thank you for reply.
Who are they? Is there nobody to ask for?

**Comment by stephenwithav** [comment]
They are the Cloud at Cost team.

You can open a ticket with support, but don't expect a quick response.  They recently "upgraded" their control panel, which broke some features of the API.

**Comment by ermaker** [comment]
Oh, I got it.
Thanks for your kindly reply.

2016년 1월 21일 (목) 오전 10:47, Steven Edwards <notifications@github.com>님이 작성:

**Comment by makefu** [comment]
@ermaker @stephenwithav any updates on this issue? i am unable to create new machines via the API which is quite unfortunate. maybe you were lucky enough to get a response from the CaC team?

**Comment by stephenwithav** [comment]
No such luck, @makefu.  Some features of the API, like `listservers`, `listtemplates`, and `delete`, work, while others like `build` don't.  I'm not quite sure why.

**Comment by makefu** [comment]
@stephenwithav T_T i also have a ticket open but no response yet , opened like 2 weeks ago.

**Comment by makefu** [comment]
@stephenwithav i just found the build API endpoint to be working again. Regards

**Comment by groggemans** [comment]
Tasks API is still down. I hope they fix this soon.

They should learn to communicate with there customers! Would make a world of difference.

**Comment by ptoal** [comment]
They are a small team, trying to do big things.  Personally, I oscillate between wanting to help them succeed, and being pissed off that they keep taking on more customers without stabilizing the service for the existing ones.

**Comment by groggemans** [comment]
@ptoal same here!

**Comment by AndreasGassmann** [comment]
Any updates on this?

Do you guys also get a valid success response when trying to create a new server, but then nothing happens?

**Comment by makefu** [comment]
@AndreasGassmann yes, same results. it seems the api degraded even more in the last weeks :(

**Comment by groggemans** [comment]
Such a shame. I was working on an Ansible module to interact with the API. Hope this will be fixed (soon).
